### PR TITLE
Sidebar UI Update, Remedied VAM Module Pages & Soft Deletion Implementation

### PR DESCRIPTION
**Pull request by biellamariscotes**
### Overview
This PR addresses several issues and improvements in the VAM module:

- 8ebd6933a96a671efddcfc0c359429da20d7e1ed **Fixed Broken Pages:** Resolved issues caused by changes in database data values.
- d9ccc0a0814745d50502769f92975949720c1508 **Soft Deletion Implementation:** Changed the deletion process of verified accounts from hard deletion to soft deletion with `account_status` of `invalid`, moving them to the Recycle Bin.
- cbe73a2a1ac35ce1655c29eff66a3cbe4b3187be **UI Updates:** Updated the sidebar and profile dropdown (addition of Recycle Bin) to the new UI design.

<hr>

### Artifacts

**⭐ New Sidebar & User Dropdown**
- Removed **Archive**, **Reports** and replaced it with **Election Reports**.
- Appended icons on User Dropdown.
- Added **Recycle Bin** dropdown item.

Updated Sidebar
<br>
**⭐ Soft Deletion of Accounts**
- Account status changed to `invalid`. Still present on database, and is not yet permanently deleted.

https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/135019439/b9b5225f-479d-4b1d-b831-32ddd2ecc73c

### CODE REVIEW

File: src/manage-voters.php
Comment (Carl-Tabuso) [inline src/manage-voters.php:1]: Minor issue: Excess double quotations on style and class attributes. Please do the needed changes.

image

image

image

File: src/submission_handlers/move-to-trashbin.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+require_once '../includes/classes/db-connector.php';
+require_once '../includes/session-handler.php';
+require_once '../includes/classes/session-manager.php';
+require_once '../includes/classes/query-handler.php';
```
Comment (Carl-Tabuso) [inline src/submission_handlers/move-to-trashbin.php]: Normalize file path traversal

**Review by Carl-Tabuso — COMMENTED** [review]
I've found many issues with voters' account validation. I would like to ask if this branch will address all of them. We can proceed with the merging and you can address these issues on your next PR. As such, I will open new issue tickets so we can track these changes.

**Found Issues:**
1. File path traversal normalization
2. No prepared statement for querying the database
3. Excessed double quotation marks on style and class attribute (see comments)
4. Button state for approving and rejecting an account.
5. Redirection to "Approve" and "Reject" page although that user has been validated already.

For 4 and 5

https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162773681/3b214967-1d83-4b4f-b58b-5ea1bd6aa98f

https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162773681/33a71866-ed69-453c-b1d7-8b6dd170e888

**Comment by biellamariscotes** [comment]
> I've found many issues with voters' account validation. I would like to ask if this branch will address all of them. We can proceed with the merging and you can address these issues on your next PR. As such, I will open new issue tickets so we can track these changes.
> 
> **Found Issues:**
> 
> 1. File path traversal normalization
> 2. No prepared statement for querying the database
> 3. Excessed double quotation marks on style and class attribute (see comments)
> 4. Button state for approving and rejecting an account.
> 5. Redirection to "Approve" and "Reject" page although that user has been validated already.
> 
> For 4 and 5
> 
>  2024-05-28.11-00-57.mp4 
>  2024-05-28.11-01-28.mp4

The following issues (1, 3, 5) have been resolved with the corresponding commit hashes:

**1.** 1a3ef9e - File path traversal normalization. 

**3.** 3377b7b - Excessed double quotation marks on style and class attribute

**5.** eefc9c0 - Redirection to "Approve" and "Reject" page although that user has been validated already.


For issues 2 & 4, I will be addressing them on my next PR. A constraint preventing access to the Validate Voter page after an account has been validated, along with the refactoring of the database query code.

Issue tickets for these pending issues would be appreciated. Thank you.

**Review by Carl-Tabuso — APPROVED** [review]
New sidebar and header contents are urgently needed by other developers so I'll just be opening new issue tickets for comments that haven't been addressed in this PR.

**Pull request merged**